go/fixchain/fix_error.go

```diff
@@ -14,18 +17,20 @@ const (
 	ParseFailure
 	CannotFetchURL
 	FixFailed
+	PostFailed
 	LogPostFailed
 	VerifyFailed
 )
 
 // FixError is the struct with which errors in the fixing process are reported
 type FixError struct {
-	Type  errorType
+	Type  errorType           // The type of error
 	Cert  *x509.Certificate   // The supplied leaf certificate
 	Chain []*x509.Certificate // The supplied chain
 	URL   string              // URL, if a URL is involved
 	Bad   []byte              // The offending bytes, if applicable
```

Comment:
The way we currently use this field it is.  However, the whole FixError struct and its fields are exported, so if someone wanted to randomly use our struct and put something other than a cert in to these bytes, they could.